### PR TITLE
DataLoader support not auto collate batch

### PR DESCRIPTION
**Pull request by heavengate**
### PR types
Function optimization

### PR changes
APIs

### Describe
DataLoader support not auto collating batch when `batch_size` is set as `None`

屏幕快照 2020-11-11 下午3 32 51

### CODE REVIEW

File: python/paddle/fluid/reader.py [hunk 1]
```diff
@@ -163,6 +163,21 @@ class DataLoader(object):
 
     For :code:`batch_sampler` please see :code:`paddle.io.BatchSampler`
 
+    **Disable automatic batching**
+
+    In certain cases such as some NLP task, instead of automatic batching,
```
Comment (chenwhql) [inline python/paddle/fluid/reader.py]: `some task` -> `some tasks`?

**Review by chenwhql — COMMENTED** [review]
Others LGTM

File: python/paddle/fluid/reader.py (hunk 1 above)
Comment (heavengate) [reply python/paddle/fluid/reader.py]: Done, thanks!

**Review by chenwhql — APPROVED** [review]

**Review by saxon-zh — APPROVED** [review]
LGTM

**Pull request merged**